### PR TITLE
Hacking md changes

### PR DESCRIPTION
**Pull request by jaidevmane**

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

File: HACKING.md [hunk 1]
```diff
@@ -17,7 +17,23 @@ Create a `Velero` Custom Resource to install Velero
 $ oc create -n oadp-operator-system -f config/samples/oadp_v1alpha1_velero.yaml
 ```
 
+## Local Development Environment
+You can test your changes by creating your own images and running your builds locally
```
Comment (shubham-pampattiwar) [inline HACKING.md:21]: @jaidevmane Thanks for adding these instructions. Can you please add to changes ?
1. Small note regarding the `podman build` command to be executed from the root directory of the operator
2. Replace `golang-operator` tag with placeholder of <IMG_TAG>

**Review by shubham-pampattiwar — CHANGES_REQUESTED** [review]

**Review by pranavgaikwad — APPROVED** [review]

File: HACKING.md (hunk 1 above)
Comment (jaidevmane) [reply HACKING.md:21]: Updated in latest commit. Thanks for the suggestions!

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**